front/src/applications/editor/tools/pointEdition/components/PointEditionLayers.tsx

```diff
@@ -39,7 +40,10 @@ export const BasePointEditionLayers = ({
     state: { nearestPoint, mousePosition, entity, objType },
     editorState: { editorLayers },
   } = useContext(EditorContext) as ExtendedEditorContextType<PointEditionState<EditorEntity>>;
-  const { mapStyle, layersSettings, issuesSettings } = useSelector(getMap);
+  const { issuesSettings } = useSelector(getMap);
+  const {
+    mapSettings: { mapStyle, layersSettings, showIGNBDORTHO },
+  } = useSelector(getEditorState);
```

Comment:
You can get issuesSettings from EditorState now :) 
```suggestion
  const {
    issuesSettings,
    mapSettings: { mapStyle, layersSettings, showIGNBDORTHO },
  } = useSelector(getEditorState);
```